symbols/arm9.yml

```diff
@@ -11799,6 +11834,16 @@ arm9:
         Each entry has the same structure as an entry in SCRIPT_VARS.
         
         type: struct script_local_var_table
+    - name: EVENT_FLAG_EXPANSION_ERROR
+      address:
+        EU: 0x209D4AC
+        NA: 0x209CF28
+        JP: 0x209E2FC
+      length:
+        EU: 0x20
+        NA: 0x20
+        JP: 0x20
+      description: Error log message for an unrecognized operation in CalcScriptVariables.
```

Comment:
The type is uint8, so is this a message or an error code?

Reply:
It's a message. I added the missing array syntax to the type.